src/terraform/command_helpers.go

```diff
@@ -148,3 +132,12 @@ func getDependencyServiceEnvVars(deployConfig deploy.Config, serviceConfig types
 	}
 	return result
 }
+func extractDependencyData(dependencyName, dependencyType string, deployConfig deploy.Config) (string, error) {
```

Comment:
Thoughts on naming this `GetDependencyEnvVariables`?

Reply:
We already have a similarly named `compileDependencyVars`, to which `extractDependencyData` is a helper. Don't you think the latter is a more clear name of what the function is doing?

Reply:
`compileDependencyVars` is getting the env vars for all dependencies. Thoughts on renaming that to  `compileDependenciesVars` (dependency becomes plural) and renaming `extractDependencyData` to `compileDependencyVar`? I think `extractDependencyData` isn't a good name because the function is doing more than that.

Reply:
I modified the `extractDependencyData` so that that's solely what it's doing - since dependency data can be used for more than just env vars I think it's apt to keep that name. I also moved all terraform flag/env var related things back up to `compileDependencyVars` since that logic is more specific these command helpers.